Haxl/Core/Exception.hs

```diff
@@ -72,7 +72,9 @@ module Haxl.Core.Exception (
   tryWithRethrow,
   ) where
 
+#if MIN_VERSION_GLASGOW_HASKELL(8, 8, 3, 0)
```

Comment:
We prefer using `#if __GLASGOW_HASKELL__ >= 808` instead.
As this is defined by ghc and it works even we are not using cabal to build.

Reply:
thinks for advice